src/js/view/view.js

```diff
@@ -201,7 +201,7 @@ function View(_api, _model) {
 
         _logo = new Logo(_model);
         _logo.setup();
-        _logo.setContainer(_playerElement);
+        _logo.setContainer(_playerElement.getElementsByClassName('jw-wrapper')[0]);
```

Comment:
there is a `_wrapperElement`. No need to query it again. See https://github.com/jwplayer/jwplayer/pull/3134/files#diff-adecb551656b12ea38bd16aad3077585R60